docs/prerequisites.md

```diff
@@ -1,93 +1,91 @@
+For handling `CredentialsRequests`, the cloud credential operator utility, [`ccoctl`](https://docs.openshift.com/container-platform/latest/authentication/managing_cloud_provider_credentials/cco-mode-sts.html#cco-ccoctl-configuring_cco-mode-sts), can be utilized.
 If you prefer not to use `ccoctl`, or your system doesn't support it, the AWS CLI can be an alternative.
 
-#### Option 1. Using `ccoctl`
+### Option 1. Using `ccoctl`
 
 1. [Extract and prepare the `ccoctl` binary](https://docs.openshift.com/container-platform/4.13/authentication/managing_cloud_provider_credentials/cco-mode-sts.html#cco-ccoctl-configuring_cco-mode-sts)
 
-2. Create AWS Load Balancer Operator's namespace:
+2. Use the `ccoctl` tool to create a IAM role from the operator's `CredentialsRequest`:
 
     ```bash
-    oc create namespace aws-load-balancer-operator
-    ```
-
-3. Use the `ccoctl` tool to process the operator's `CredentialsRequest` objects needed to bootstrap the operator:
-
-    ```bash
-    curl --create-dirs -o <path-to-credrequests-dir>/cr.yaml https://raw.githubusercontent.com/openshift/aws-load-balancer-operator/main/hack/operator-credentials-request.yaml
-    ccoctl aws create-iam-roles \
+   $ curl --create-dirs -o <path-to-credrequests-dir>/cr.yaml https://raw.githubusercontent.com/openshift/aws-load-balancer-operator/main/hack/operator-credentials-request.yaml
+   $ CCOCTL_OUPUT=$(mktemp)
+   $ ccoctl aws create-iam-roles \
         --name <name> --region=<aws_region> \
         --credentials-requests-dir=<path-to-credrequests-dir> \
-        --identity-provider-arn <oidc-arn>
+        --identity-provider-arn <oidc-arn> 2>&1 | tee "${CCOCTL_OUPUT}"
+
+    2023/09/12 11:38:57 Role arn:aws:iam::777777777777:role/<name>-aws-load-balancer-operator-aws-load-balancer-operator created
+    2023/09/12 11:38:57 Saved credentials configuration to: /home/user/credrequests-dir/manifests/aws-load-balancer-operator-aws-load-balancer-operator-credentials.yaml
+    2023/09/12 11:38:58 Updated Role policy for Role <name>-aws-load-balancer-operator-aws-load-balancer-operator created
     ```
 
     For each `CredentialsRequest` object, `ccoctl` creates an IAM role with a trust
     policy that is tied to the specified OIDC identity provider, and permissions
     policy as defined in each `CredentialsRequest` object. This also generates a set
-    of secrets in a **manifests** directory that is required
-    by the **aws-load-balancer-operator**.
+    of secrets in a `manifests` directory, which are not needed by the operator.
 
-4. Apply the secrets to your cluster:
+3. Write down the role ARN for the aws-load-balancer-operator from the output of `ccoctl`:
 
     ```bash
-    ls manifests/*-credentials.yaml | xargs -I{} oc apply -f {}
+    $ ROLEARN=$(\grep -Po 'arn:aws:iam[0-9a-z/:\-_]+' "${CCOCTL_OUPUT}")
```

Comment:
```suggestion
    $ ROLEARN=$(grep -Po 'arn:aws:iam[0-9a-z/:\-_]+' "${CCOCTL_OUTPUT}")
```

I also have a slight issue with this current automation. Per my other comments, I placed `--credentials-requests-dir=<path-to-credrequests-dir>` into my existing credentials request dir created by `ccoctl` during installation.

This works fine, but re-applies the credentials and the output contains all of the RoleARNs, which would break this as it would grab all of the IAM roleARNs.

Maybe it was a noob move on my part? Or should we provide a hard-coded `<path-to-credrequests-dir>` to imply it's new and not it's supposed to be an existing directory?

Reply:
`\grep` was to prevent the usage of the bash aliases, for instance, I use a grep alias which may give a different output. But for sake brevity, I agree it's not necessary.

> Or should we provide a hard-coded <path-to-credrequests-dir> to imply it's new and not it's supposed to be an existing directory?

I see. Well, yes I think there are a lot of ways of how the end user may screw up these commands. I updated the step to:
```
3. Extract and verify the operator's role ARN from the output of `ccoctl` command:                                       

    ```bash
    $ ROLEARN=$(grep -Po 'arn:aws:iam[0-9a-z/:\-_]+' "${CCOCTL_OUTPUT}")
    $ echo "${ROLEARN}"
    arn:aws:iam::777777777777:role/<name>-aws-load-balancer-operator-aws-load-balancer-operator
    ```
```
in order to delegate some responsibilities to the end user :)

Reply:
Oh wow, TIL about avoiding bash aliases like that.

Okay, that works for me, thanks :+1: